<launch>
  <node
    pkg="rsi_lauv_ntnu"
    type="bridgeConnection"
    name="bridgeConnection"
    output="screen"
  >
    <param
      name="server_addr"
      type="str"
      value="127.0.0.1"
    />
    <param
      name="server_port"
      type="str"
      value="30100"
    />
    <param
      name="vehicle_name"
      type="str"
      value="asv-jetyaksim-thor"
    />
  </node>
  <node
    pkg="rsi_lauv_ntnu"
    type="verboseLauvNode"
    name="verboseLauvNode_0"
    output="screen">
  </node>
  <node
    pkg="g2s_interface"
    type="fakeRgi_node"
    name="fakeRgi_node"
    output="screen">
  </node>
</launch>
<!-- launch-prefix="xterm -e" -->
